### DISCUSSION TITLE
Song Transcriptor App

### DISCUSSION THREAD

**Discussion by BartekKrzepkowski** [openai/whisper#206] [Show and tell]
Hi,
Few days ago I created application, in streamlit, which is using GPT-3 and Whisper model that transcript song and can provide some information about well known songs.

App link:
https://bartekkrzepkowski-song-transcriptor-app-streamlit-app-e3htiq.streamlitapp.com/
GitHub link:
https://github.com/BartekKrzepkowski/Song_Transcriptor_App

If someone has ideas how to develop this app even further it would be great.

Best regards

**Comment by moebiussurfing** [comment]
hello, sorry bc the offtopic,
but did you tryied to get the word time marks?
I mean instead of full sentence default.
that could be useful also to some karaoke stuff.
thanks.

**Reply by BartekKrzepkowski** [reply]
It is a great idea.
Is there any functionality to track every single word or do you mean some algorithm that I can't see right now, right of the bat?

**Comment by moebiussurfing** [comment]
I was asking if the Whisper API allows tweaking like that.
